src/Oro/Bundle/CacheBundle/Provider/ConfigCacheWarmerInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Oro\Bundle\CacheBundle\Provider;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+interface ConfigCacheWarmerInterface
+{
+    /**
+     * @param ContainerBuilder $containerBuilder
```

Comment:
add a comment to describe the what this method should do
it I understand correct this method should add resources that are used to build the given container builder in this container builder